lib/types/local-time.js

```diff
@@ -27,257 +29,221 @@ const nanoSecInMillis = Long.fromNumber(1000000);
  * @private
  * */
 const millisInDay = 86400000;
+
 /**
- *
- * Creates a new instance of LocalTime.
  * @class
  * @classdesc A time without a time-zone in the ISO-8601 calendar system, such as 10:30:05.
```

Comment:
``@class`` and ``@classdesc`` not needed
https://jsdoc.app/howto-es2015-classes